src/Symfony/Component/Console/Input/InputOption.php

```diff
@@ -21,10 +21,25 @@
  */
 class InputOption
 {
+    /**
+     * Do not accept input for the option (e.g. --yell). This is the default behavior of options.
+     */
     public const VALUE_NONE = 1;
```

Comment:
Last one :) There should be a blank line between each constant, as per our coding standards

Reply:
thanks, fixed in followup commit